client/cmd/validator.go

```diff
@@ -151,7 +152,7 @@ type genPrivKeyJSONConfig struct {
 }
 
 func loadEnv() {
-	if err := godotenv.Load("../.env"); err != nil {
```

Comment:
This is rolled back. This was due to my local debug env (used `/client` as working dir), so `.env` under the root should be fine by default.